torch/onnx/_internal/exporter/_torchlib/ops/core.py

```diff
@@ -45,3 +45,60 @@ def aten_add_complex(self: TReal, other: TReal, alpha: float = 1.0) -> TReal:
     """add.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor"""
 
     return aten_add(self, other, alpha=alpha)
+
+
+@onnx_impl((aten.grid_sampler.default), trace_only=True)
+def aten_grid_sampler(
+    input: TReal,
+    grid: TReal,
+    interpolation_mode: int,
+    padding_mode: int,
+    align_corners: bool,
+) -> TReal:
+    """grid_sampler(Tensor input, Tensor grid, int interpolation_mode, int padding_mode, bool align_corners) -> Tensor"""
+
+    inter_mode_options = ("bilinear", "nearest", "bicubic")
+    inter_mode_str = inter_mode_options[interpolation_mode]
+
+    padding_mode_options = ("zeros", "border", "reflection")
+    padding_mode_str = padding_mode_options[padding_mode]
+
+    # Only one onnx Op so don't put into private function
+    return op.GridSample(
+        input,
+        grid,
+        align_corners=align_corners,
+        mode=inter_mode_str,
+        padding_mode=padding_mode_str,
+    )
+
+
+@onnx_impl((aten.grid_sampler.default), trace_only=True, opset_introduced=20)
+def aten_grid_sampler(
+    input: TReal,
+    grid: TReal,
+    interpolation_mode: int,
+    padding_mode: int,
+    align_corners: bool,
+) -> TReal:
+    """grid_sampler(Tensor input, Tensor grid, int interpolation_mode, int padding_mode, bool align_corners) -> Tensor"""
+
+    inter_mode_options = ("bilinear", "nearest", "bicubic")
+    inter_mode_str = inter_mode_options[interpolation_mode]
+    if inter_mode_str == "bilinear":
+        inter_mode_str = "linear"
+    elif inter_mode_str == "bicubic":
+        inter_mode_str = "cubic"
+
+    padding_mode_options = ("zeros", "border", "reflection")
+    padding_mode_str = padding_mode_options[padding_mode]
+
+    # Only one onnx Op so don't put into private function
+    from onnxscript.onnx_opset import opset20 as op20
```

Comment:
I would move import to top of the file, also

```suggestion
    from onnxscript import opset20 as op20
```